### ISSUE TITLE
Introduce Documentation 2.0

### ISSUE THREAD

**Issue by mikenikles** [gitpod-io/website#461]
### What's happening?
We are working on new documentation to provide a more streamlined onboarding experience while also simplifying other educational aspects such as concepts, references and troubleshooting.

### What to expect
* A rewrite of the existing docs - we won't leave a stone untouched
* To minimize disruption, we are going to keep the existing docs and all URLs in place while we write the new docs
* A banner on all current docs pages to let you know where to find the new docs in case you want to check them out
* Continuous releases of new docs pages

### Feedback welcome
If you have any feedback, we'd absolutely love to hear it - for the existing as well as the new docs. The easiest way to provide feedback is by commenting below. You could also open a new issue or comment directly on pull requests when they're open 🙏.

**Comment by mteam88** [comment]
I don't think this should be closed. There is still a link to this issue here: https://www.gitpod.io/docs/references/gitpod-dockerfile#custom-docker-image.

**Comment by pawlean** [comment]
> I don't think this should be closed. There is still a link to this issue here: https://www.gitpod.io/docs/references/gitpod-dockerfile#custom-docker-image.

Hi @mteam88! Thanks for reporting 🎯 I've just opened a new issue (#1854) to track this.
